unime/src/i18n/en/index.ts

```diff
@@ -189,26 +194,64 @@ const en = {
       DATA: 'Data',
       BADGES: 'Badges',
     },
+    CREDENTIAL: {
+      NAVBAR_TITLE: 'Credential info',
+      NAME: '',
+    },
+    BADGE: {
+      NAVBAR_TITLE: 'Badge info',
+      DESCRIPTION: 'Description',
+      METADATA: 'Metadata',
+    },
     EMPTY_CREDENTIALS: {
       TITLE: "It's a bit quiet in here",
       SUBTITLE: 'Why not add some credentials to start your new digital me?',
     },
     FAVORITES: 'My favorites',
   },
   ACTIVITY: {
+    BOTTOM_NAVIGATION_TITLE: 'Activity',
     NAVBAR_TITLE: 'Connected',
     TABS: {
       CONNECTIONS: 'Connections',
       TIMELINE: 'Timeline',
     },
   },
+  SCAN: {
+    BOTTOM_NAVIGATION_TITLE: 'Scan',
+    TITLE: 'Scan a',
```

Comment:
Please follow the pattern `TITLE_1`, `TITLE_2` for multiline text (instead of `TITLE`, `TITLE1`).